modules/cluster-compare-exclude-metadata.adoc

```diff
@@ -0,0 +1,138 @@
+:_mod-docs-content-type: PROCEDURE
+
+[id="cluster-compare-exclude-metadata_{context}"]
+= Configuring the metadata.yaml file to exclude template fields
+
+You can configure the `metadata.yaml` file to exclude fields from a comparison. Exclude fields that are irrelevant to a comparison, for example annotations or labels that are inconsequential to a cluster configuration. 
+
+You can configure exclusions in the `metadata.yaml` file in the following ways:
+
+* Exclude all fields in a custom resource not specified in a template.
+
+* Exclude specific fields that you define using the `pathToKey` field.
++
+[NOTE]
+====
+`pathToKey` is a dot separated path. Use quotes to escape key values featuring a period.
+====
+
+[id="cluster-compare-ignore-all-fields_{context}"]
+== Excluding all fields not specified in a template
+
+During the comparison process, the `cluster-compare` plugin renders a template by merging fields from the corresponding custom resource (CR). If you configure the `ignore-unspecified-fields` to `true`, all fields that are present in the CR, but not in the template, are excluded from the merge. Use this approach when you want to focus the comparison on the fields specified in the template only.
+
+.Procedure
+
+* Create a `metadata.yaml` file to match your use case. Use the following structure as an example:
++
+[source,yaml]
+----
+apiVersion: v2
+parts:
+  - name: Part1
+    components:
+      - name: Namespace
+        allOf:
+          - path: namespace.yaml
+            config:
+              ignore-unspecified-fields: true <1>
+#...
+----
+<1> Specify `true` to exclude from the comparison all fields in a CR that are not explicitly configured in the corresponding `namespace.yaml` template.
+
+[id="cluster-compare-ignore-specified-fields_{context}"]
+== Excluding specific fields
+
+You can specify fields to exclude by defining the path to the field, and then referencing the definition in the `config` section for a template. 
+
+.Procedure
+
+* Create a `metadata.yaml` file to match your use case. Use the following structure as an example:
++
+.Example metadata.yaml file
+[source,yaml]
+----
+apiVersion: v2
+parts:
+    - name: Part1
+      components: 
+        - name: Component1
+            - path: deployment.yaml
+              config:
+                fieldsToOmitRefs:
+                    - deployments <1>
+
+#...
+
+fieldsToOmit:
+   items:
+      deployments:
+         - pathToKey: spec.selector.matchLabels.k8s-app <2>
+----
+<1> References the `deployments` item for the `deployment.yaml` template.
+<2> Excludes the `spec.selector.matchLabels.k8s-app` field from the comparison. 
++
+[NOTE]
+====
+Setting `fieldsToOmitRefs` replaces the default value.
```

Comment:
I don't think we mention the default value, so it's hard to know what good this note will do...

Reply:
Swapped the order as suggested, thanks!